### PR TITLE
Add flipper and conditional WIP-page for 21-10210

### PR DESCRIPTION
**Pull request by tlei123**
## Summary

- Add Flipper feature-flag for Lay/Witness Statement (21-10210), along with conditional WIP-page [for when feature-flag is OFF, or ON but not 100%].
- Team: VA Product Forms
- Companion [back-end PR](https://github.com/department-of-veterans-affairs/vets-api/pull/12904) already merged.
- Companion [content-build final-URL PR](https://github.com/department-of-veterans-affairs/content-build/pull/1604) opened.

## Related issue(s)

- _Link to ticket created in va.gov-team-forms repo_
department-of-veterans-affairs/va.gov-team-forms#307

## Testing done

- Locally tested successfully with mocked feature-flag on & off.

### CODE REVIEW

**Review by gmrabian — APPROVED** [review]
Tested out the changes locally and they look good!
Screenshot 2023-06-13 at 9 11 34 AM

I think the margin between the header and the WIP component will be bigger once we have breadcrumbs – see note in content-build PR (maybe we can add them now?)

**Review by randimays — APPROVED** [review]
LGTM

**Review by rsmithadhoc — APPROVED** [review]

File: src/applications/static-pages/simple-forms/21-10210/App.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+const App = ({ formEnabled }) => {
+  if (formEnabled === undefined) {
+    return <va-loading-indicator set-focus message="Loading..." />;
+  }
+
+  if (formEnabled) {
+    return (
+      <>
+        <p>You can apply online or by mail.</p>
```
Comment (gmrabian) [inline src/applications/static-pages/simple-forms/21-10210/App.js:16]: Small content change here:
`You can submit this form online or by mail.` per the [content doc](https://docs.google.com/document/d/1n5rrOrAmdmj9NavOpR8fbjQgFV88iEV4/edit)

**Review by gmrabian — CHANGES_REQUESTED** [review]
sorry just noticed this

File: src/applications/static-pages/simple-forms/21-10210/App.js (hunk 1 above)
Comment (tlei123) [reply src/applications/static-pages/simple-forms/21-10210/App.js:16]: Fixed.  Thx for catching this!

**Review by gmrabian — CHANGES_REQUESTED** [review]

File: src/applications/static-pages/simple-forms/21-10210/App.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ const App = ({ formEnabled }) => {
   if (formEnabled) {
     return (
       <>
-        <p>You can apply online or by mail.</p>
+        <p>You can submit the form online or by mail.</p>
```
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/static-pages/simple-forms/21-10210/App.js]: Fixed.  Sorry!  :D

**Review by gmrabian — APPROVED** [review]

File: src/applications/static-pages/simple-forms/21-10210/App.js (hunk 1 above)
Comment (tlei123) [reply src/applications/static-pages/simple-forms/21-10210/App.js]: fixed.

File: src/applications/static-pages/simple-forms/21-10210/App.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tlei123) [reply src/applications/static-pages/simple-forms/21-10210/App.js]: fixed.

**Pull request merged**